app/controllers/users/sessions_controller.rb

```diff
@@ -55,7 +55,13 @@ def track_authentication_attempt(email)
 
     def cache_active_profile
       cacher = Pii::Cacher.new(current_user, user_session)
-      cacher.save(current_user.user_access_key)
+      begin
+        cacher.save(current_user.user_access_key)
+      rescue Pii::EncryptionError => _err
+        current_user.active_profile.deactivate
+        properties = { user_id: current_user.uuid }
+        analytics.track_event(Analytics::PROFILE_ENCRYPTION_INVALID, properties)
```

Comment:
Right, I think it might be worthwhile logging the exception to NewRelic? I don't have a sense of how much we'll be looking at analytics vs exceptions, they're kind of similar but not